src/main/java/org/broadinstitute/consent/http/models/dataset_registration_v1/DatasetRegistrationSchemaV1.java

```diff
@@ -6,12 +6,16 @@
 import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import com.fasterxml.jackson.annotation.JsonValue;
+import org.broadinstitute.consent.http.enumeration.DatasetPropertyType;
+import org.broadinstitute.consent.http.models.DatasetProperty;
+import org.broadinstitute.consent.http.util.gson.GsonUtil;
 
 import javax.annotation.processing.Generated;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
nit: The added imports are unused

Reply:
no way, i'm gonna find that setting

Reply:
haha, wait sorry, I should have also given you the setting...
<img width="984" alt="image" src="https://user-images.githubusercontent.com/5633787/214949449-ac24d2f3-be70-4b25-a07a-19e67ace57a8.png">


Note the "optimize imports on the fly"

Reply:
Still an issue